src/pilot/config_idrac.py

```diff
@@ -657,7 +657,7 @@ def config_idrac(instack_lock,
                     new_drac_client.is_idrac_ready()
                     password_changed = True
                 except exceptions.WSManInvalidResponse as ex:
-                    if "unauthorized" in ex.message.lower():
+                    if "unauthorized" in str(ex).lower():
```

Comment:
Why the change here?  Is WSMan changing?

Reply:
Python3 handling or exceptions changed